server/s3_object_metadata.cc

```diff
@@ -670,6 +670,10 @@ int S3ObjectMetadata::from_json(std::string content) {
   account_name = system_defined_attribute["Owner-Account"];
   account_id = system_defined_attribute["Owner-Account-id"];
   object_version_id = system_defined_attribute["x-amz-version-id"];
+
+  if (object_version_id.empty()) {
```

Comment:
when version_id could be empty ?

We saved the object metadata with empty version ID, in previous PUT object call  ? @knrajnambiar76, @DPG17  !!

Reply:
yes, but we need to understand why it was empty ? Is there some issue with PUT request !!

Reply:
object_version_id, in this from_json() function, should never be empty.

Reply:
We need to investigate why object_version_id is empty.

Reply:
@knrajnambiar76 @DPG17 @amitwac2608 
M.b. something is incomplete while initializing multipart upload?